oc_cdtapi/tests/test_BitbucketAPI.py

```
@@ -0,0 +1,266 @@
+import logging
+import os
+import re
+import unittest
+
+import sys
+if sys.version_info >= (3, 3):
```

Ruff (0.15.12)

[warning] Version block is outdated for minimum Python version

Remove outdated version block

(UP036)